assets/panel/debugger.properties

```diff
@@ -166,9 +136,9 @@ sourceSearch.search.key=F
 # the search for re-searching the same search triggered from a sourceSearch
 sourceSearch.search.again.key=G
 
-# LOCALIZATION NOTE (sourceSearch1.resultsSummary): Shows a summary of
+# LOCALIZATION NOTE (sourceSearch.resultsSummary): Shows a summary of
 # the number of matches for autocomplete
-sourceSearch1.resultsSummary=%d instances of “%S”
+sourceSearch.resultsSummary=%d instances of “%S”
```

Comment:
Argh, forget this. I have no idea why, but I thought I was commenting on a different string (the one with %s vs %S). Too many tabs open I guess :-\